server/routes/aws/tasks.py

```
@@ -0,0 +1,134 @@
+        "summary": f"Security finding detected: {title}. Desc: {desc}",
+        "risk_level": severity,
+        "suggested_fix": suggested_fix
+    }
+
+@shared_task
+def process_securityhub_finding(payload: dict, org_id: str):
+    """
+    Background task to process Security Hub finding webhook payloads.
+    Generates AI triage context and upserts records to PostgreSQL.
+    """
+    logger.info(f"[SECURITY_HUB] Processing background task for event {payload.get('id')}")
```

Ruff (0.15.12)

[warning] Logging statement uses f-string

(G004)